### PR TITLE
[ID] Update Bot Account

### PR DESCRIPTION
**Pull request by AncuL001**

### CODE REVIEW

File: wiki/Bot_account/id.md [hunk 1]
```diff
@@ -5,61 +5,58 @@ tags:
...
+Akun bot hanya dapat dibuat melalui [proses pengajuan](#mendaftarkan-akun-bot). Upaya membuat akun bot dengan mendaftarkan akun osu! seperti biasa akan dianggap sebagai memiliki banyak akun dan merupakan pelanggaran terhadap [peraturan komunitas](/wiki/Rules#peraturan-komunitas).
 
 ## Keuntungan akun bot
 
-Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [*chat*](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan lebih tinggi supaya layanan mereka dalam berinteraksi dengan pengguna yang lebih banyak dapat berjalan tanpa risiko akun terkena [pembungkaman akun](/wiki/Glossary#silence).
+Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [obrolan](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan yang lebih tinggi supaya layanan mereka dalam berinteraksi dengan pengguna yang lebih banyak tanpa risiko membuat akun tersebut [dibungkam](/wiki/Glossary#silence).
```
Comment (AncuL001) [inline wiki/Bot_account/id.md:18]:
```suggestion
Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [obrolan](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan yang lebih tinggi supaya layanan mereka dalam berinteraksi dengan lebih banyak pengguna tanpa risiko membuat akun tersebut [dibungkam](/wiki/Glossary#silence).
```

File: wiki/Bot_account/id.md [hunk 2]
```diff
@@ -5,61 +5,58 @@ tags:
...
 
 ### Mengajukan permintaan
 
-Jika bot kamu memenuhi kriteria yang tertera di atas, kamu dapat melakukan pengajuan untuk dibuatkan akun khusus bot dari bot yang kamu buat.
+Jika bot kamu memenuhi kriteria yang tertera di atas, kamu dapat melakukan pengajuan pembuatan akun bot untuk bot tersebut.
 
 Silakan mengirim email ke [accounts@ppy.sh](mailto:accounts@ppy.sh) dengan judul `Bot Account Request`. Email ini harus dikirim dari alamat email yang tersambung dengan akun osu! kamu.
 
-Isi daripada email tersebut harus meliputi hal - hal berikut:
+Badan email tersebut harus meliputi hal - hal berikut:
```
Comment (AncuL001) [inline wiki/Bot_account/id.md:46]:
```suggestion
Badan email tersebut harus meliputi hal-hal berikut:
```

File: wiki/Bot_account/id.md [hunk 3]
```diff
@@ -5,61 +5,58 @@ tags:
...
+Akun bot hanya dapat dibuat melalui [proses pengajuan](#mendaftarkan-akun-bot). Upaya membuat akun bot dengan mendaftarkan akun osu! seperti biasa akan dianggap sebagai memiliki banyak akun dan merupakan pelanggaran terhadap [peraturan komunitas](/wiki/Rules#peraturan-komunitas).
 
 ## Keuntungan akun bot
 
-Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [*chat*](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan lebih tinggi supaya layanan mereka dalam berinteraksi dengan pengguna yang lebih banyak dapat berjalan tanpa risiko akun terkena [pembungkaman akun](/wiki/Glossary#silence).
+Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [obrolan](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan yang lebih tinggi supaya layanan mereka dalam berinteraksi dengan lebih banyak pengguna tanpa risiko membuat akun tersebut [dibungkam](/wiki/Glossary#silence).
```
Comment (hikikuman) [inline wiki/Bot_account/id.md:18]:
```suggestion
Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [obrolan](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan yang lebih tinggi supaya layanan mereka dalam berinteraksi dengan lebih banyak pengguna tanpa risiko membuat akun tersebut [di-silence](/wiki/Glossary#silence).
```
`silence` shouldn't be translated to `bungkam`.

File: wiki/Bot_account/id.md [hunk 4]
```diff
@@ -5,61 +5,58 @@ tags:
...
-- Bot harus digunakan oleh setidaknya 50 pengguna berbeda setiap bulannya
-- Bot mematuhi batasan tarif pesan yang dimiliki akun pribadi
-- Bot tidak boleh mengirim pesan apa pun di *channel* publik
-- Bot harus digunakan demi kepentingan dan keuntungan komunitas.
+- Bot telah berjalan di akun kamu selama setidaknya 6 bulan
+- Bot bersifat sepenuhnya *open source* dan memiliki dokumentasi publik lengkap
+- Bot digunakan oleh setidaknya 50 pengguna berbeda setiap bulannya
+- Bot mematuhi batasan pengiriman pesan yang dimiliki akun pribadi
+- Bot tidak mengirim pesan apa pun di *channel* publik
+- Bot digunakan demi kepentingan dan keuntungan komunitas.
```
Comment (hikikuman) [inline wiki/Bot_account/id.md:38]:
```suggestion
- Bot digunakan demi kepentingan dan keuntungan komunitas
```

**Review by hikikuman — CHANGES_REQUESTED** [review]

**Comment by TicClick** [comment]
@hikikuman is this good to go?

File: wiki/Bot_account/id.md [hunk 5]
```diff
@@ -5,61 +5,58 @@ tags:
...
+Akun bot hanya dapat dibuat melalui [proses pengajuan](#mendaftarkan-akun-bot). Upaya membuat akun bot dengan mendaftarkan akun osu! seperti biasa akan dianggap sebagai memiliki banyak akun dan merupakan pelanggaran terhadap [peraturan komunitas](/wiki/Rules#peraturan-komunitas).
 
 ## Keuntungan akun bot
 
-Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [*chat*](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan lebih tinggi supaya layanan mereka dalam berinteraksi dengan pengguna yang lebih banyak dapat berjalan tanpa risiko akun terkena [pembungkaman akun](/wiki/Glossary#silence).
+Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [obrolan](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan yang lebih tinggi supaya layanan mereka dalam berinteraksi dengan lebih banyak pengguna tanpa risiko membuat akun tersebut [di-silence](/wiki/Glossary#silence).
```
Comment (hikikuman) [inline wiki/Bot_account/id.md:18]:
```suggestion
Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [obrolan](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan yang lebih tinggi supaya layanan mereka dalam berinteraksi dengan lebih banyak pengguna tanpa risiko membuat akun tersebut [di-silence](/wiki/Glossary#silence):
```

File: wiki/Bot_account/id.md [hunk 6]
```diff
@@ -5,61 +5,58 @@ tags:
...
+Akun bot hanya dapat dibuat melalui [proses pengajuan](#mendaftarkan-akun-bot). Upaya membuat akun bot dengan mendaftarkan akun osu! seperti biasa akan dianggap sebagai memiliki banyak akun dan merupakan pelanggaran terhadap [peraturan komunitas](/wiki/Rules#peraturan-komunitas).
 
 ## Keuntungan akun bot
 
-Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [*chat*](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan lebih tinggi supaya layanan mereka dalam berinteraksi dengan pengguna yang lebih banyak dapat berjalan tanpa risiko akun terkena [pembungkaman akun](/wiki/Glossary#silence).
+Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [obrolan](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan yang lebih tinggi supaya layanan mereka dalam berinteraksi dengan lebih banyak pengguna tanpa risiko membuat akun tersebut [di-silence](/wiki/Glossary#silence):
```
Comment (AncuL001) [inline wiki/Bot_account/id.md:18]:
```suggestion
Perbedaan utama antara akun pribadi dan akun bot terdapat pada batas pengiriman pesan dalam [obrolan](/wiki/Chat_Console). Akun bot memiliki batas pengiriman pesan yang lebih tinggi supaya layanan mereka dapat berinteraksi dengan lebih banyak pengguna tanpa risiko membuat akun tersebut [di-silence](/wiki/Glossary#silence):
```

**Review by hikikuman — APPROVED** [review]
Checked.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**